app/assets/stylesheets/cats.css.scss

```diff
@@ -0,0 +1,58 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+
+.container#main-container {
+  margin-top: 80px;
```

Comment:
Properties should be ordered margin-left, margin-right, margin-top, padding-left, padding-right